src/util/ssa_expr.cpp

```diff
@@ -31,8 +31,8 @@ static void build_ssa_identifier_rec(
 
     build_ssa_identifier_rec(member.struct_op(), l0, l1, l2, os, l1_object_os);
 
-    os << '.' << member.get_component_name();
-    l1_object_os << '.' << member.get_component_name();
+    os << ".." << member.get_component_name();
+    l1_object_os << ".." << member.get_component_name();
```

Comment:
This should be documented somewhere. In particular is the documentation of `build_ssa_identifier_rec` still correct?

Reply:
Documentation updated, thanks for spotting this!